manifests/dropin_file.pp

```diff
@@ -4,8 +4,8 @@
 #
 # @see systemd.unit(5)
 #
-# @param unit the The target unit file to create, the value will be set to the `filename` parameter as well
-# @param filename The target unit file to create
+# @param unit the The target unit file to create
+# @param filename The target unit file to create, the value will default to the name of the type
```

Comment:
Following through about my previous comment, perhaps it should be something like:
```suggestion
# @param filename The filename of the drop in. The full path is determined using the path, unit and this filename.
```